Client/imfeelinglucky.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
This HTML page is not needed because you would be repeating the results.html page. 

If you could find a way to use JS to adapt the results.html depending on what you input into the search bar, this could save you a lot of work.